explore/realspace.py

```diff
@@ -636,7 +637,7 @@ def calc_Iq_magnetic(qx, qy, rho, rho_m, points, volume=1.0, view=(0, 0, 0),
     yaw and roll for a beam travelling along the negative z axis.
     *up_frac_i* is the portion of polarizer neutrons which are spin up.
     *up_frac_f* is the portion of analyzer neutrons which are spin up.
-    *up_theta* and *up_phi* are the rotation angle of the spin up direction 
+    *up_theta* and *up_phi* are the rotation angle of the spin up direction
     in the detector plane and the inclination from the beam direction (z-axis).
```

Comment:
Order is reversed: theta is the inclination from the beam direction and phi is the rotation in the detector plance